src/frontend/src/lib/components/hero/about/AboutWhatModal.svelte

```diff
@@ -23,18 +22,12 @@
 			<Html text={replaceOisyPlaceholders($i18n.about.what.text.hold_crypto)} />
 		</p>
 
-		<p class="mt-6">
-			<IconIncognito />
-			<Html text={replaceOisyPlaceholders($i18n.about.what.text.go_incognito)} />
-		</p>
-
 		<p class="mt-6">
 			<IconWalletConnect />
 			<Html text={replaceOisyPlaceholders($i18n.about.what.text.use_eth_dapps)} />
 		</p>
 
-		<!-- TODO: Remove the opacity when it will be possible to use IC dapps with Oisy-->
```

Comment:
Removing this TODO is related to this PR?

Reply:
Correct, they changed idea: we can already remove the opacity, independently of having already the feature or not